### DISCUSSION TITLE
Astriod framework insert module in article

### DISCUSSION THREAD

**Discussion by anilvc31** [templaza/astroid-framework#370] [Q&A] [answered]
Sir,

I am using Astroid framework in my website https://www.vinoibabhve.org.
Kindly guide me, on how to insert the module in an article.
Thank you,
Anil V. Chaudhary

**Comment by deltapapa01** [comment]
Hi @anilvc31,
this is a joomla feature, you can use the joomla docs to add an module to an article:
https://docs.joomla.org/How_do_you_put_a_module_inside_an_article%3F

WBR,
deltapapa

@Chacapamac: I don't think this is an issue, but is better left in the discussion section. Maybe you can move this?

**Comment by sonvnn** [comment, marked as answer]
Moved to discusstion. 
@anilvc31 you can use loadmodule feature of Joomla Content. 

1. Go to Edit an article -> Click "CMS Content" -> Select "Module"
2. A modal will appear -> You can choose your module here

![Screen Shot 2023-08-09 at 15 04 48](https://github.com/templaza/astroid-framework/assets/3204103/ceffe0f1-bd54-4c5e-b894-30a51b49109f)

Hope it help!
